src/main/java/coffee/client/feature/command/impl/VClip.java

```diff
@@ -16,7 +16,7 @@
 
 public class VClip extends Command {
     public VClip() {
-        super("VClip", "Teleport vertically", "vclip");
+        super("vclip", "Teleport vertically", "vclip <amount>");
```

Comment:
the first arg is the command name, not the trigger. the 3rd are the triggers, this will break the command